### PR TITLE
Fixing wrong link in README

### PR DESCRIPTION
**Pull request by rachelnabors**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ with sync_playwright() as playwright, playwright.chromium.launch(headless=False)
 - [Save and reuse logged in state](https://github.com/tinyfish-io/agentql/tree/main/examples/save_and_load_authenticated_session)
 - [Run AgentQL in Stealth Mode](https://github.com/tinyfish-io/fish-tank/tree/main/examples/stealth_mode)
```
Comment (colriot) [inline README.md:131]:
```suggestion
- [Run AgentQL in Stealth Mode](https://github.com/tinyfish-io/agentql/tree/main/examples/stealth_mode)
```

File: README.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ with sync_playwright() as playwright, playwright.chromium.launch(headless=False)
...
 - [Wait for page to load](https://github.com/tinyfish-io/agentql/tree/main/examples/wait_for_entire_page_load)
-- [Collect pricing data from an e-commerce website using AgentQL](https://github.com/tinyfish-io/fish-tank/tree/main/examples/close_cookie_dialog)
+- [Collect pricing data from an e-commerce website using AgentQL](https://github.com/tinyfish-io/agentql/tree/main/application_examples/collect_ecommerce_pricing_data)
 - [Perform sentiment analysis](https://github.com/tinyfish-io/fish-tank/tree/main/application_examples/perform_sentiment_analysis)
 - [Get an element's XPath](https://github.com/tinyfish-io/fish-tank/tree/main/application_examples/xpath)
```
Comment (colriot) [inline README.md:135]:
```suggestion
- [Perform sentiment analysis](https://github.com/tinyfish-io/agentql/tree/main/application_examples/perform_sentiment_analysis)
- [Get an element's XPath](https://github.com/tinyfish-io/agentql/tree/main/application_examples/xpath)
```

File: README.md [hunk 3]
```diff
@@ -130,7 +130,7 @@ with sync_playwright() as playwright, playwright.chromium.launch(headless=False)
...
 - [Run script online in Google Colaboratory](./examples/run_script_online_in_google_colab)
```
Comment (colriot) [inline README.md:136]: I think this (using relative links) might actually be a better idea in terms of being prone to renames, etc.

**Review by colriot — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (rachelnabors) [reply README.md:136]: Thanks for catching these! You make a good case. I endorse it for a future update

**Pull request merged**